src/index.html

```diff
@@ -2,11 +2,354 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+          name="viewport">
+    <meta content="ie=edge"
+          http-equiv="X-UA-Compatible">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com"
+          rel="preconnect">
```

Comment:
```suggestion
    <link href="https://fonts.googleapis.com" rel="preconnect">
```
if element has 2 attributes you can leave them on the same line